Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: The error message indicates the radio field is required. With radio `input` elements, you can set `required` on each `input` and browsers will only require a user to choose one of them.
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
Error: A `button` is not part of this component. Note: If radio inputs are wrapped in their `label`, `br` is unnecessary.
diff --git a/output/Radio Group with Validation Error/3/d971055b-87f4-46c7-a868-9031d3f96e50.html b/output/Radio Group with Validation Error/3/d971055b-87f4-46c7-a868-9031d3f96e50.html
index 527f27d..25843f1 100644
--- a/output/Radio Group with Validation Error/3/d971055b-87f4-46c7-a868-9031d3f96e50.html	
+++ b/output/Radio Group with Validation Error/3/d971055b-87f4-46c7-a868-9031d3f96e50.html	
@@ -1,17 +1,20 @@
-<form>
-  <fieldset>
-    <legend>Choose a theme:</legend>
-    <label>
-      <input type="radio" name="theme" value="light"> Light
-    </label><br>
-    <label>
-      <input type="radio" name="theme" value="dark"> Dark
-    </label><br>
-    <label>
-      <input type="radio" name="theme" value="high_contrast"> High Contrast
-    </label>
-  </fieldset>
-  <span id="error" style="color: red; display: none;">You must choose a theme to proceed</span>
-  <br>
-  <input type="submit" value="Submit">
-</form>
\ No newline at end of file
+<fieldset aria-describedby="error">
+  <legend>Choose a theme:</legend>
+  <span id="error">You must choose a theme to proceed</div>
+  <label for="light">
+    <input type="radio" id="light" name="theme" value="light"
+           required aria-invalid="true">
+    Light
+  </label>
+  <label for="dark">
+    <input type="radio" id="dark" name="theme" value="dark"
+           required aria-invalid="true">
+    Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" id="high-contrast" name="theme"
+           value="high-contrast"
+           required aria-invalid="true">
+      High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
